code/modules/cooking/machinery/cooking_machines/oven.dm

```diff
@@ -122,6 +122,24 @@
 		return
 	..()
 
+/obj/machinery/appliance/cooker/oven/small
+	name = "compact oven"
+	desc = "A lightweight, small oven. Doesn't hold much, but it cooks just fine."
+	density = 0
+	anchored = 0
```

Comment:
it's designed to be unanchored?

Reply:
It's basically the size of a microwave so yeah